pkg/kubectl/cmd/patch.go

```diff
@@ -100,6 +100,7 @@ func NewCmdPatch(f cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringP("patch", "p", "", "The patch to be applied to the resource JSON file.")
 	cmd.MarkFlagRequired("patch")
 	cmd.Flags().String("type", "strategic", fmt.Sprintf("The type of patch being provided; one of %v", sets.StringKeySet(patchTypes).List()))
+	cmd.Flags().Bool("all", false, "Select all resources in the namespace of the specified resource types")
```

Comment:
Allowing the user to patch `all` resources in a namespace increases the change of the user blow up their cluster.
I'm not blocking this. But please add an example for it.